syntax = "proto3";

package alimama.proto;

message SliceRequest {
    uint64 slice_partition = 1;
    uint64 data_start = 2;
    uint64 data_len = 3;
}

message Request {
  repeated SliceRequest slice_request = 1;
}

message Response {
  int32 status = 1; // 0，返回结果ok，-1 服务发生内部错误
  repeated bytes slice_data = 2;
}

service ModelService {
  rpc Get(Request) returns (Response) {}
}